cmd/cli/serve/serve.go

```diff
@@ -247,14 +251,19 @@ func serve(cmd *cobra.Command, cfg types.Bacalhau, fsRepo *repo.FsRepo) error {
 	}
 
 	if !cfg.DisableAnalytics {
-		installationID, err := fsRepo.ReadInstallationID()
+		installationID, err := metastore.ReadInstallationID()
 		if err != nil {
 			log.Trace().Err(err).Msg("failed to read installationID")
 		}
+		instanceID, err := metastore.ReadInstanceID()
+		if err != nil {
+			log.Trace().Err(err).Msg("failed to read instanceID")
+		}
```

Comment:
trace are mainly for more granular and request level logging. It might be useful to use debug level here

Reply:
changing to trace